### PR TITLE
Add expressions

### PR DESCRIPTION
**Pull request by graichen**
built on top of moveWhitespace pull request.

### CODE REVIEW

File: Day1/Expressions.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+str1 = "abc"     
+str2 = "defghij"
+strlong = str1 + str2    
+```
+
+### Conversion operations
+
+Sometimes you need to convert a variable from one data type to another. We saw that python will figure out the type by the assignment. If you need to change it you use a conversion function. For instance, to convert from a string to an integer, use the int() function.
+
+Some common and interesting conversion functions:
```
Comment (sarcar) [inline Day1/Expressions.md:32]: image
 Table not rendered properly (tested on Linux/Chrome)

Comment (sarcar) [reply Day1/Expressions.md:32]: What you probably want is this:

Some common and interesting conversion functions:

| Function | Behavior |
|----|----|
| int | convert to an integer |
| float | convert to a real number |
| str | convert to a string (for printing or concatenating with other strings) |
| hex | convert to a string in hexadecimal format prefixed by 0x |
| ord | return the integer code for a single character |
| chr | return the character for a given unicode integer value |

Two things that you may want to take care of:
a) Github markdown requires that you leave a blank line before the table for proper rendering
b) The "|---|---|" construct should be immediately after your header row (usually the second row)

**Review by sarcar — CHANGES_REQUESTED** [review]
The section on Expressions is very well-written. Suggest a minor modification to get the markdown table to render properly on github

File: Day1/Expressions.md (hunk 1 above)
Comment (sarcar) [reply Day1/Expressions.md:32]: Here's the markdown that works

```
Some common and interesting conversion functions:
<empty line here>
| Function | Behavior |
|----|----|
| int | convert to an integer |
| float | convert to a real number |
| str | convert to a string (for printing or concatenating with other strings) |
| hex | convert to a string in hexadecimal format prefixed by 0x |
| ord | return the integer code for a single character |
| chr | return the character for a given unicode integer value |
```

will give you:

Some common and interesting conversion functions:

| Function | Behavior |
|----|----|
| int | convert to an integer |
| float | convert to a real number |
| str | convert to a string (for printing or concatenating with other strings) |
| hex | convert to a string in hexadecimal format prefixed by 0x |
| ord | return the integer code for a single character |
| chr | return the character for a given unicode integer value |

Comment (graichen) [reply Day1/Expressions.md:32]: that's the problem with learning a language by  imitation - sometimes you don't get they nuances right. I had been checking the visibility in Chrome after pushing to the branch before submitting the pull request - I guess I got distracted & forgot to check this one. Thanks for the great help.

All fixed now...

Note this PR includes moveWhitespace...

**Review by sarcar — APPROVED** [review]

**Comment by sarcar** [comment]
Thanks Cathy. I will let you delete the used branches if you choose to.

**Pull request merged**